source/organization/contributors.html.md

```diff
@@ -25,9 +25,30 @@ Skills:
 - Tech-related: Rust, JavaScript, HTML, CSS/SCSS, Linux
 - Languages: French (native), English (not perfect), German (not good), Esperanto (still learning)
 
+# Trinity/fdb-hiroshima
+
+**Member of the moderation team**
+
+Preferred pronouns: they/theme
+
+Open for mentoring: no, but feel free to ask questions when you need
+
+Accounts:
+
+- GitHub: fdb-hiroshima
+- Matrix: @fdb_hiroshima:matrix.org
+- Fediverse: @trinity@soc.punktrash.club (Pleroma), @FDB_hiroshima@fediverse.blog (Plume)
+- Loomio: @fdbhiroshima
+- Crowdin: fdb-hiroshima
+
+Skills:
+
+- Tech-related: Rust, Linux
```

Comment:
I have no idea what my skills are, but they are definitely not front-end related

Reply:
SQL, python, shell